.github/workflows/nodejs.yml

```diff
@@ -36,5 +39,17 @@ jobs:
       run: npm i
 
     - name: Test
-      run: npm test
+      id: test
+      run: |
+        npm run test-coverage-ci || echo "exit_code=$?" >> $GITHUB_OUTPUT
+
+    - name: Comment test coverage report on PR
+      uses: romeovs/lcov-reporter-action@v0.2.16
+      with:
+        lcov-file: ./coverage/lcov.info
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Exit if coverage condition not met
+      if: ${{ steps.test.outputs.exit_code }} != 0
+      run: exit ${{ steps.test.outputs.exit_code }}
```

Comment:
@maoo - thoughts on these changes?

Reply:
The context is that I've combined recording coverage and covered lines limit checking within the test step to avoid re-running all tests. The next step is to report the findings as a PR comment. Storing the exit code in the test step is essential because if the codebase doesn't meet the required criteria, it errors out. However, we need it to first report the findings as a PR comment, so I store the exit code and pass it to the last step. The workflow then exits and prevents the PR from merging.

You can see how this behaves and what data is presented at CyberCitizen01#1 (A test PR) and the [workflow run](https://github.com/CyberCitizen01/git-proxy/actions/runs/5707493190/job/15464227432).

Reply:
@CyberCitizen01 - thanks for the detailed description 👍